miasm2/analysis/depgraph.py

```diff
@@ -759,15 +759,13 @@ def _follow_mem_wrapper(exprs, mem_read):
             follow.update(expr.get_r(mem_read=mem_read, cst_read=True))
         return follow, set()
 
-    @staticmethod
-    def _follow_mem(exprs):
+    def _follow_mem(self, exprs):
```

Comment:
`influencegraph` subclass `depgraph` , so a call to `copy` or `extend` in incluence graph will generate a depgraph. As we need history and self, this the `copy` and `extend` cannot be a class method.